.github/workflows/node-ci-pr.yml

```diff
@@ -161,26 +161,3 @@ jobs:
           pattern: "**/dist/es/*.js"
           # Always ignore SourceMaps and node_modules:
           exclude: "{**/*.map,**/node_modules/**}"
-
-  build_storybook:
```

Comment:
**note:** Found out this step was redundant as this is already done in `chromatic-build.yml`.